prowler/providers/aws/services/codebuild/codebuild_service.py

```diff
@@ -84,12 +84,21 @@ def _batch_get_projects(self, project):
             if project_info["source"]["type"] != "NO_SOURCE":
                 project.source = Source(
                     type=project_info["source"]["type"],
-                    location=project_info["source"]["location"],
+                    location=(
+                        project_info["source"]["location"]
+                        if project_info["source"].get("location") is not None
+                        else ""
+                    ),
```

Comment:
You can also do `project_info["source"].get("location","") to have it in one line without the `if/else`